janMarkusMilan/assignments/slidingwindow/lc53/lc53.js

```diff
@@ -1,25 +1,29 @@
+// Problem Statement #
+
+// Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
+
```

Comment:
It would be beneficial to your interviewer and yourself to psuedo-code your approach to this problem in several sentences.